src/main/java/com/gluonhq/impl/maps/ImageRetriever.java

```diff
@@ -61,7 +62,10 @@
 
     static {
         try {
-            File storageRoot = PlatformFactory.getPlatform().getPrivateStorage();
+            File storageRoot = Services.get(StorageService.class)
+                    .flatMap(StorageService::getPrivateStorage)
+                    .orElseThrow(() -> new IOException());
```

Comment:
Exception needs a good description. Something like "Storage Service is not available"